packages/docs/src/config/theme-config.ts

```diff
@@ -14,29 +14,29 @@ export const COLOR_THEMES: Record<ThemeName, Record<string, string>> = {
   [ThemeName.DEFAULT]: {
     primary: '#2550C0',
     secondary: '#767C88',
-    success: '#40e583',
-    info: '#2c82e0',
+    success: '#40E583',
+    info: '#2C82E0',
     danger: '#DE1041',
     warning: '#FFAC0A',
 
-    backgroundPrimary: '#ffffff',
```

Comment:
Can't we use lowercase colors?

In all other places it's lowercase

Reply:
```ts
{
 danger: '#f0f',
 'background-primary': '#fff
}
```

It doesn't look too nice. BTW, it is not case-sensitive.

Reply:
@m0ksem meant hexes 😄, so `#ffac0a` instead of `#FFAC0A`.